libs/ledger-live-common/src/hooks/useBroadcast.ts

```diff
@@ -15,7 +14,7 @@ type SignTransactionArgs = {
 export const useBroadcast = ({ account, parentAccount }: SignTransactionArgs) => {
   const broadcast = useCallback(
     async (signedOperation: SignedOperation): Promise<Operation> => {
-      invariant(account, "account not present");
+      if (!account) throw new Error("account not present");
```

Comment:
Any specific reason not to keep using `invariant` here?

Also, nitpick but:
```suggestion
      if (!account) {
        throw new Error("account not present");
      }
```

Reply:
Typescript is still complaining with `invariant` when we consider `account` as valid in the next lines.

Reply:
👍  yes. it's a good idea to get rid of `invariant` for this reason.